flowzone.yml

```diff
@@ -469,8 +469,16 @@
     with:
       version: v0.14.0
 
+  - &setupPython # https://github.com/actions/setup-python
+    name: Setup python
+    id: setup-python
+    uses: actions/setup-python@61a6322f88396a6271a6ee3565807d608ecaddd1 # v4
+    with:
+      python-version: "3.9"
```

Comment:
Just wondering, who is to decide the appropriate version here? Do we go with the latest that is known to work?

Reply:
This default value is only used by `python_publish` and `python_finalize` and I'm not sure what effect a new version would have. IMO it would be best to keep it relatively recent but I didn't want to introduce a change like that in this PR.

Reply:
Makes sense, thanks.